ui/controllers/settings_controller.py

```
@@ -97,3 +137,205 @@ def show_general_settings_window(self):
+                        controller_ref.set('channel', login)
+                    controller_ref.save_config()
+
+                    self.send_response(200)
+                    self.end_headers()
+                    self.wfile.write(b"ok")
+
+                    def _shutdown():
+                        try:
+                            controller_ref._oauth_server_httpd.shutdown()
+                        except Exception:
+                            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)